File: nodejs/7 kyu/151.js
// https://www.codewars.com/kata/5a084a098ba9146690000969/train/javascript

// Passed

function timeConvert(num) {
  if (num < 0) {
    num = 0;
  }

  let applyPadding = (num) =>
    String(num).length === 1 ? "0" + String(num) : String(num);
  let hours = applyPadding(Math.floor(num / 60));
  let minutes = applyPadding(num % 60);
  let convertedTime = [hours, minutes].join(":");
  return convertedTime;
}

const output = timeConvert(75);
console.log(output);

File: python/7 kyu/151.py
# https://www.codewars.com/kata/59afff65f1c8274f270020f5/train/python

# Passed

def spinning_rings(inner_max, outer_max):
    inner = inner_max
    outer = 1
    
    move_count = 1
    while inner != outer:
        inner -= 1
        inner = inner_max if inner < 0 else inner 

        outer += 1
        outer = 0 if outer > outer_max else outer
        
        move_count += 1

    return move_count

output = spinning_rings(2, 3)
print(output)